.travis.yml

```diff
@@ -154,24 +140,18 @@ jobs:
   allow_failures:
     - <<: *linter-stage
 
-install:
-  - ccache --max-size=1G
-  - COMMAND="make -C src minisat2-download" &&
-    eval ${PRE_COMMAND} ${COMMAND}
-  - COMMAND="make -C src/ansi-c library_check" &&
-    eval ${PRE_COMMAND} ${COMMAND}
-  - COMMAND="make -C src CXX=\"$COMPILER\" CXXFLAGS=\"-Wall -Werror -pedantic -O2 -g $EXTRA_CXXFLAGS\" -j2" &&
-    eval ${PRE_COMMAND} ${COMMAND}
-  - COMMAND="make -C src CXX=\"$COMPILER\" CXXFLAGS=\"$FLAGS $EXTRA_CXXFLAGS\" -j2 clobber.dir memory-models.dir musketeer.dir" &&
-    eval ${PRE_COMMAND} ${COMMAND}
+install: |
```

Comment:
This will probably convert it to `command1; command2; command3` which will probably do not catch if `command1` or `command2` fail. Can you (@reuk) keep it as it was before, please? Or prove me wrong.